test/e2e/template/helmfile/testdata/snapshot/templated_lockfile/input.yaml

```diff
@@ -6,7 +6,6 @@ environments:
   prod:
   staging:
 
----
```

Comment:
Can we keep this snapshot as-is and create a new snapshot testdata for testing this new policy?